packages/@o3r/configuration/schematics/configuration-to-component/index.ts

```diff
@@ -31,6 +31,28 @@ import { basename, dirname } from 'node:path';
 import * as ts from 'typescript';
 import type { NgAddConfigSchematicsSchema } from './schema';
 
+const configProperties = [
+  'dynamicConfig$', 'config', 'config$'
+];
+
+const hasConfiguration = (path: string, tree: Tree) => {
+  const o3rClassDeclaration = ts.createSourceFile(
```

Comment:
that doesn't look optimal to use `createSourceFile` multiple times
shouldn't we pass the sourceFile directly as a parameter ?